content/applications/sales/sales/amazon_connector/update.rst

```diff
@@ -2,24 +2,17 @@
 Install the Amazon Connector Authentication Update
 ==================================================
 
-Starting July 2020, the Amazon Connector requires the use of a new authentication
-method that makes it easier for you to set up the Connector.
+Starting July 2022, Amazon change it's API and the Amazon Connector needs to be updated to continue
+to exchange information with Amazon. This also makes it easier for you to set up the Connector.
 
-Prior to the update, Odoo customers who wished to use the Amazon Connector had
-to apply to get developer credentials through Amazon, which was a painful and long
-process that could take weeks. This method of authentication is still technically
-possible, but Amazon will refuse to provide developer credentials for Odoo customers
-from July 2020 onwards and will deactivate such credentials for existing customers.
-
-This documentation will help you install the update and use the new authentication
-flow.
+This documentation will help you install the update and use the new authentication flow.
 
 Note that if your Odoo database was first created after the update was released, the
 update module is installed automatically. You can check if this module is already installed
-by going to the  **Apps** menu, removing the ``Apps`` search facet and search for ``amazon``.
-If the module **Amazon/Authentication Patch** is present and marked as installed, your Odoo
-database is already up-to-date and you can proceed with the :doc:`setup <setup>` step
-of the Amazon Connector.
+by going to the :guilabel:`Apps` menu, removing the ``Apps`` search facet and search for ``amazon``.
```

Comment:
Reworking this sentence (continued from the last comment) and fixing some Odoo terminology.
```suggestion
go to :guilabel:`Apps`, remove the ``Apps`` filter and search for ``Amazon``.
```